src/utils/strings.ts

```diff
@@ -190,3 +190,26 @@ export const stripAndremoveHeadingsFromHTML = (content: string) => {
 
   return string;
 };
+
+/**
+ * To be used specifically for token symbols.
+ *
+ * If a token symbol exceeds 5 characters, it truncates the symbol
+ * to 5 characters followed by an ellipsis.
+ * *
+ * @method truncateTokenSymbol
+ *
+ * @param {string} tokenSymbol The token symbol to conditionally truncate
+ *
+ * @return {string} If the string is more than 5 characters i.e. TOKENSYMBOL, it returns "TOKEN..."
+ * Otherwise, it will return what was passed.
+ */
+export const truncateTokenSymbol = (tokenSymbol?: string) => {
+  if (tokenSymbol) {
+    return tokenSymbol.length > 5
+      ? `${tokenSymbol.slice(0, 5)}...`
+      : tokenSymbol;
+  }
+
+  return tokenSymbol;
+};
```

Comment:
Why not make this generic, and configurable via a argument. We really don't need this to be so specific

Reply:
@rdig it was only because I couldn't come up with another use case for it in the codebase and wanted the function to be very "oddly specific" about token length 👀 but yes you're right! There's nothing stopping me from making this more generic 😄 Will push some changes!

Reply:
Hey @rdig I've removed that function and I'm now using the existing string truncation function. All pushed 🚀